docs/src/modules/components/Ad.js

```diff
@@ -5,8 +5,8 @@ import { withStyles } from '@material-ui/core/styles';
 import Typography from '@material-ui/core/Typography';
 import Tooltip from '@material-ui/core/Tooltip';
 import Paper from '@material-ui/core/Paper';
-import CodeFund from 'docs/src/modules/components/CodeFund';
-import Carbon from 'docs/src/modules/components/Carbon';
+import AdCodeFund from 'docs/src/modules/components/AdCodeFund';
+import AdCarbon from 'docs/src/modules/components/AdCarbon';
```

Comment:
`CodeFundAd`, `CarbonAd`?

Reply:
I wanted to quickly edit the two files. They were not showing side by side in my folder view.

Reply:
CodeFundAd is more grammatically correct.